### ISSUE TITLE
Error: ImageWindow.windowById(): invalid argument type: String expected.

### ISSUE THREAD

**Issue by r4um** [jarmoruuth/AutoIntegrate#16]
Fails with v1.4.3, I think it was working fine before this update.

```
2022-01-23 11:43:26] Writing file:
[2022-01-23 11:43:26] C:/Users/eligo/astro/TelescopeLivee_NGC_2359/auto/AutoProcessed/Integration_O.xisf
[2022-01-23 11:43:26] Writing image: w=3056 h=3056 n=1 Gray Float32
[2022-01-23 11:43:26] 110 FITS keyword(s) embedded.
[2022-01-23 11:43:26] 16 image properties embedded.
[2022-01-23 11:43:26] Error: ImageWindow.windowById(): invalid argument type: String expected.
[2022-01-23 11:43:26] Processing stopped!
[2022-01-23 11:43:26] Write processing steps to C:/Users/eligo/astro/TelescopeLivee_NGC_2359/auto/AutoProcessed/AutoIntegrate.log

-- CUT--

PixInsight version 1.8.8-12
AutoIntegrate v1.43
Processing options:
Fix column defects true
Fix row defects true
Integrate only true
Start processing...
CreateChannelImages
calibrateEngine, no bias, flat or dark files
run Linear Defect Detection
Image LDD integration on 18 files
  Using AdditiveWithScaling for ImageIntegration normalization
run CosmeticCorrection for linear defect file group 0, 18 files
run CosmeticCorrection, output *_cc.xisf, number of line defects to fix is 123
runSubframeSelector
SSWEIGHT 5.940186277918079, FWHM 2.6392058839735073, Ecc 0.6007768476481672, SNR 1.3887957200217518, Stars 906, PSFSignal 4.940186277918079, PSFPower 865.6798659892175, C:/Users/eligo/astro/TelescopeLivee_NGC_2359/auto/AutoOutput/2022-01-14T01-09-12_NGC_2359_Halpha_T-25_600s_cal_cc.xisf
SSWEIGHT 7.295109085201156, FWHM 2.316769550982777, Ecc 0.5537944410151824, SNR 1.3790261006504123, Stars 1021, PSFSignal 6.295109085201156, PSFPower 937.8241906967213, C:/Users/eligo/astro/TelescopeLivee_NGC_2359/auto/AutoOutput/2022-01-14T01-20-47_NGC_2359_Halpha_T-25_600s_cal_cc.xisf
SSWEIGHT 3.919204163384026, FWHM 2.3787926150575314, Ecc 0.5115671180041079, SNR 1.3283587313281597, Stars 720, PSFSignal 2.919204163384026, PSFPower 436.4561897417685, C:/Users/eligo/astro/TelescopeLivee_NGC_2359/auto/AutoOutput/2022-01-15T03-19-48_NGC_2359_Halpha_T-25_600s_cal_cc.xisf
SSWEIGHT 5.1008564401862655, FWHM 2.9507087396588703, Ecc 0.5059114498932782, SNR 1.363949573502801, Stars 762, PSFSignal 4.1008564401862655, PSFPower 821.2145980352018, C:/Users/eligo/astro/TelescopeLivee_NGC_2359/auto/AutoOutput/2022-01-15T03-31-26_NGC_2359_Halpha_T-25_600s_cal_cc.xisf
SSWEIGHT 7.65080064372107, FWHM 3.990831737056764, Ecc 0.4493700471239667, SNR 1.5872302487749188, Stars 741, PSFSignal 6.65080064372107, PSFPower 2363.5272785372294, C:/Users/eligo/astro/TelescopeLivee_NGC_2359/auto/AutoOutput/2022-01-19T05-45-27_NGC_2359_Halpha_T-25_600s_cal_cc.xisf
SSWEIGHT 8.490782250299098, FWHM 3.471639029727473, Ecc 0.4505240495362229, SNR 1.5973523223220785, Stars 884, PSFSignal 7.490782250299099, PSFPower 2230.0120114553292, C:/Users/eligo/astro/TelescopeLivee_NGC_2359/auto/AutoOutput/2022-01-19T05-56-39_NGC_2359_Halpha_T-25_600s_cal_cc.xisf
SSWEIGHT 13.274379965399092, FWHM 2.3882860973644515, Ecc 0.5554471206294553, SNR 1.3650024983985256, Stars 1386, PSFSignal 12.274379965399092, PSFPower 2136.5154954510117, C:/Users/eligo/astro/TelescopeLivee_NGC_2359/auto/AutoOutput/2022-01-14T01-57-11_NGC_2359_SII_T-25_600s_cal_cc.xisf
SSWEIGHT 9.90954344340616, FWHM 2.3813599700214065, Ecc 0.6046593930015577, SNR 1.3445573572761131, Stars 1181, PSFSignal 8.90954344340616, PSFPower 1487.4839943387724, C:/Users/eligo/astro/TelescopeLivee_NGC_2359/auto/AutoOutput/2022-01-14T02-09-45_NGC_2359_SII_T-25_600s_cal_cc.xisf
SSWEIGHT 12.394089883108336, FWHM 2.4873672028324734, Ecc 0.5287888774269293, SNR 1.399946008688546, Stars 1337, PSFSignal 11.394089883108336, PSFPower 2324.537402800562, C:/Users/eligo/astro/TelescopeLivee_NGC_2359/auto/AutoOutput/2022-01-16T03-22-23_NGC_2359_SII_T-25_600s_cal_cc.xisf
SSWEIGHT 12.7177297800677, FWHM 2.3489453253870596, Ecc 0.516534506802623, SNR 1.3961238505875777, Stars 1418, PSFSignal 11.7177297800677, PSFPower 2160.010919935772, C:/Users/eligo/astro/TelescopeLivee_NGC_2359/auto/AutoOutput/2022-01-16T03-33-38_NGC_2359_SII_T-25_600s_cal_cc.xisf
SSWEIGHT 11.827642527819174, FWHM 2.767586884163138, Ecc 0.48513528946035844, SNR 1.3938571630253382, Stars 1286, PSFSignal 10.827642527819174, PSFPower 2561.6676667831593, C:/Users/eligo/astro/TelescopeLivee_NGC_2359/auto/AutoOutput/2022-01-19T03-17-02_NGC_2359_SII_T-25_600s_cal_cc.xisf
SSWEIGHT 11.93411624026365, FWHM 2.9939465012455457, Ecc 0.47300088943320107, SNR 1.4045318070863464, Stars 1206, PSFSignal 10.93411624026365, PSFPower 2850.8237991044252, C:/Users/eligo/astro/TelescopeLivee_NGC_2359/auto/AutoOutput/2022-01-19T03-29-24_NGC_2359_SII_T-25_600s_cal_cc.xisf
SSWEIGHT 6.448799945074879, FWHM 2.201558424123432, Ecc 0.5283359499372735, SNR 1.4108308380488983, Stars 964, PSFSignal 5.448799945074879, PSFPower 834.817076810722, C:/Users/eligo/astro/TelescopeLivee_NGC_2359/auto/AutoOutput/2022-01-14T01-31-55_NGC_2359_OIII_T-25_600s_cal_cc.xisf
SSWEIGHT 10.220906981968295, FWHM 2.146505735843742, Ecc 0.5097711822105424, SNR 1.46141290304773, Stars 1080, PSFSignal 9.220906981968295, PSFPower 1556.4381208872223, C:/Users/eligo/astro/TelescopeLivee_NGC_2359/auto/AutoOutput/2022-01-14T01-44-20_NGC_2359_OIII_T-25_600s_cal_cc.xisf
SSWEIGHT 8.675293642062362, FWHM 2.0200555829433435, Ecc 0.6064021427330079, SNR 1.4263761639006174, Stars 1071, PSFSignal 7.675293642062362, PSFPower 1144.646685806605, C:/Users/eligo/astro/TelescopeLivee_NGC_2359/auto/AutoOutput/2022-01-15T04-32-05_NGC_2359_OIII_T-25_600s_cal_cc.xisf
SSWEIGHT 9.030243054348128, FWHM 2.404840481758539, Ecc 0.6114886499788057, SNR 1.5121851095090915, Stars 1110, PSFSignal 8.030243054348128, PSFPower 1569.1688219947525, C:/Users/eligo/astro/TelescopeLivee_NGC_2359/auto/AutoOutput/2022-01-16T03-10-42_NGC_2359_OIII_T-25_600s_cal_cc.xisf
SSWEIGHT 6.287760377780822, FWHM 4.0484917323178875, Ecc 0.4701344183457013, SNR 1.571826075995106, Stars 623, PSFSignal 5.287760377780822, PSFPower 2239.5941141470676, C:/Users/eligo/astro/TelescopeLivee_NGC_2359/auto/AutoOutput/2022-01-19T06-09-14_NGC_2359_OIII_T-25_600s_cal_cc.xisf
SSWEIGHT 6.092873357683339, FWHM 3.690510340089492, Ecc 0.5329162542628781, SNR 1.5662062431535593, Stars 708, PSFSignal 5.092873357683339, PSFPower 1813.6098491657137, C:/Users/eligo/astro/TelescopeLivee_NGC_2359/auto/AutoOutput/2022-01-19T06-20-30_NGC_2359_OIII_T-25_600s_cal_cc.xisf
runSubframeSelector, input 18 files, output 18 files
Find best SSWEIGHT
Star alignment on 18 files, reference image C:/Users/eligo/astro/TelescopeLivee_NGC_2359/auto/AutoOutput/2022-01-14T01-57-11_NGC_2359_SII_T-25_600s_cal_cc_a.xisf
runStarAlignment, 18 files
Find L,R,G,B,H,S,O and color channels
Processing as RGB files
Image H integration on 6 files
  Using AdditiveWithScaling for ImageIntegration normalization
  Auto2 using Percentile clip for rejection
Image S integration on 6 files
  Using AdditiveWithScaling for ImageIntegration normalization
  Auto2 using Percentile clip for rejection
Image O integration on 6 files
  Using AdditiveWithScaling for ImageIntegration normalization
  Auto2 using Percentile clip for rejection
```

Full AutoInterate.log attached
[AutoIntegrate.log](https://github.com/jarmoruuth/AutoIntegrate/files/7920419/AutoIntegrate.log)
.

**Comment by jarmoruuth** [comment]
Thanks for the bug report! I reproduced the issue. The problem was in Integrate only option. It did run the integration but incorrectly tried to save a non-existing final image. Fix pushed to the master branch.

**Comment by jarmoruuth** [comment]
Fix included in v1.44
